docs/biowulf.md

```diff
@@ -0,0 +1,28 @@
+# nf-core/configs: biowulf Configuration
+
+nf-core pipelines have been configured for use on the Biowulf cluster at the NIH.
+
+To use, run the pipeline with `-profile biowulf`, this will pull the [`biowulf.config`] (../conf/biowulf.config) from github which has been pre-configured with a setup suitable for the Biowulf cluster. By default, Nextflow will download Singularity containers to a cache directory setted by the config file. 
```

Comment:
Following the main config, users will have to go `-profile biowulf,biowulflocal` or `-profile biowulf,biowulf` to properly use it as configured.

Reply:
`biowulf,biowulf`?

Reply:
This is too complicated since we only have 1 cluster for now, I removed the local executor part from `biowulf.config`, only keep the slurm executor, I think "less is more" in this case. Thanks for point that out.